src/main/java/uk/gov/hmcts/cmc/claimstore/models/Interest.java

```diff
@@ -68,6 +68,6 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return String.format("Interest{type='%s', rate=%f, reason='%s'}", type, rate, reason);
+        return String.format("AmountContent{type='%s', rate=%f, reason='%s'}", type, rate, reason);
```

Comment:
Class name is not changed to it should be `Interest` or even better use reflection based to string.

Reply:
intellij was overly keen but will just do the reflection one